test/Atmos/EDMF/closures/mixing_length.jl

```diff
@@ -3,41 +3,35 @@
     mixing_length(
         m::AtmosModel{FT},
         ml::MixingLengthModel,
-        state::Vars,
-        diffusive::Vars,
-        aux::Vars,
-        t::Real,
+        args,
         Δ::Tuple,
         Et::Tuple,
         ts,
```

Comment:
maybe we should call the grid mean TS as "ts_gm"

Reply:
Ah, good catch. I dropped this earlier because the cache uses NamedTuple and the symbols must match (e.g, `@unpack ts = precomputed`), but we can unpack without that syntax directly into a `ts_gm` alias.